inc/compat/plugins/compat-plugin-automation-web-platform.php

```diff
@@ -0,0 +1,296 @@
+<?php
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Compatibility with plugin: Wawp - OTP Verification, Order Notifications, and Country Code Selector for WooCommerce (by Wawp).
+ */
+class FluidCheckout_WawpOTPVerification extends FluidCheckout {
```

Comment:
- Fix class name convention. Should be related to the name of the plugin folder, in this case `automation-web-platform`, the class should be called `FluidCheckout_AutomationWebPlatform`.